### PR TITLE
Improve UX template editor

### PR DESCRIPTION
**Pull request by balloob**
I was hanging out in #general when I noticed some confusion about the template dev tool. People didn't realize that the left was editable.

Changes made:
 - Change label from "template" to "Template editor"
 - Autofocus the template editor
 - Change font of editor to monospace font

### CODE REVIEW

File: panels/dev-template/ha-panel-dev-template.html [hunk 1]
```diff
@@ -121,31 +131,27 @@
...
+  "unit": "°C"
+} %}
+
+The temperature is {{ my_test_json.temperature }} {{ my_test_json.unit }}.
+
+{% if is_state("device_tracker.paulus", "home") and
+      is_state("device_tracker.anne_therese", "home") -%}
+  You are both home, you silly
+{%- else -%}
+  Anne Therese is at {{ states("device_tracker.anne_therese") }} and  +
```
Comment (c727) [inline panels/dev-template/ha-panel-dev-template.html]: remove `+` at the end

File: panels/dev-template/ha-panel-dev-template.html [hunk 2]
```diff
@@ -76,7 +82,11 @@
             <li><a href='http://jinja.pocoo.org/docs/dev/templates/' target='_blank'>Jinja2 template documentation</a></li>
             <li><a href='https://home-assistant.io/topics/templating/' target='_blank'>Home Assistant template extensions</a></li>
           </ul>
-          <paper-textarea label="Template" value='{{template}}'></paper-textarea>
+          <paper-textarea
+            label="Template Editor"
```
Comment (c727) [inline panels/dev-template/ha-panel-dev-template.html]: `Template editor`

**Review by c727 — COMMENTED** [review]
maybe we should also change the name in the dev tools panel?

File: panels/dev-template/ha-panel-dev-template.html [hunk 3]
```diff
@@ -121,31 +131,27 @@
...
+{%- else -%}
+  Anne Therese is at {{ states("device_tracker.anne_therese") }} and  +
+Paulus is at {{ states("device_tracker.paulus") }}
+{%- endif %}
+
+For loop example:
+{% for state in states.sensor -%}
+  {%- if loop.first %}The {% elif loop.last %} and the {% else %}, the {% endif -%}
+  {{ state.name | lower }} is {{state.state_with_unit}}
+{%- endfor -%}.`,
```
Comment (c727) [inline panels/dev-template/ha-panel-dev-template.html]: 2nd `-`not needed

**Comment by balloob** [comment]
All the names are describing the topic. The tools available for that topic can change over time.

**Pull request merged**